geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -196,4 +196,20 @@ public ClusterManagementResult deleteRegion(
     return new ResponseEntity<>(result,
         HttpStatus.CREATED);
   }
+
+  @ApiOperation(value = "delete region index")
+  @PreAuthorize("@securityService.authorize('CLUSTER', 'MANAGE', 'QUERY')")
+  @DeleteMapping(REGION_CONFIG_ENDPOINT + "/{regionName}" + INDEXES + "/{indexName}")
```

Comment:
Would the index name have "." in it? Should we add that :. thing like other end points?

Reply:
Good catch. We should do that for all endpoints that end with a parameter

Reply:
Updated the code to cover this scenario.